demisto_sdk/commands/common/errors.py

```diff
@@ -367,12 +370,27 @@ def invalid_v2_integration_name():
     def found_hidden_param(parameter_name):
         return f"Parameter: \"{parameter_name}\" can't be hidden. Please remove this field."
 
+    @staticmethod
+    @error_code_decorator
+    def invalid_deprecated_integration(display_name, description):
+        error_answer = ''
+        if not display_name.startswith('Deprecated.'):
+            error_answer += 'The display_name (display) of your integration should start with "Deprecated.".\n'
+        if not description.startswith('Deprecated.'):
+            error_answer += 'The description of your integration should start with "Deprecated.".\n'
+        return error_answer
+
     @staticmethod
     @error_code_decorator
     def invalid_v2_script_name():
         return "The name of this v2 script is incorrect , should be **name**V2." \
                " e.g: DBotTrainTextClassifierV2"
 
+    @staticmethod
+    @error_code_decorator
+    def invalid_deprecated_script():
+        return 'The script comment has to start with "Deprecated."'
```

Comment:
Same here - let's make these changes everywhere
```suggestion
        return 'Every deprecated script's comment has to start with "Deprecated."'
```